crypto/rc4/rc4_enc.c

```diff
@@ -13,10 +13,8 @@
 /*-
  * RC4 as implemented from a posting from
  * Newsgroups: sci.crypt
- * From: sterndark@netcom.com (David Sterndark)
  * Subject: RC4 Algorithm revealed.
  * Message-ID: <sternCvKL4B.Hyy@netcom.com>
- * Date: Wed, 14 Sep 1994 06:35:31 GMT
```

Comment:
The removed lines are kind of "fingerprint". If removed, then it would be appropriate to reword comment, maybe as "from a posting on sci.crypt newsgroup in September 1994." No?

Reply:
the message-id should be unique but I'll put the date back anyway. :)

Reply:
and also the date put back for rc4_skey